File: mrobertsdev+elixir/fizzbuzz.ex
defmodule FizzBuzz do
    def upto(n) when n>0, do: 1..n |> Enum.map(&fizzbuzz/1)
    
    defp fizzbuzz(n) when rem(n, 3) == 0 and rem(n, 5) == 0, do: "FizzBuzz"
    defp fizzbuzz(n) when rem(n, 3) == 0,                    do: "Fizz"
    defp fizzbuzz(n) when rem(n, 5) == 0,                    do: "Buzz"
    defp fizzbuzz(n),                                        do: n
end

File: tim_dunnington+kotlin/fizzbuzz.kt
fun main(args : Array<String>) {
    for(i in 1..100) {
        when {
            i%15 == 0 -> System.out.println("FizzBuzz")
            i%5 == 0 -> System.out.println("Buzz")
            i%3 == 0 -> System.out.println("Fizz")
            else -> System.out.println(i)
        }
    }
}